traffic_ops/testing/api/v5/tc-fixtures.json

```diff
@@ -341,7 +341,7 @@
         {
```

Comment:
it would seem not. The PR that added that field didn't add them to the fixtures and the tests all pass fine. I believe that's abusing the fact that if a field isn't found when unmarshaling, Go will just leave it as the "zero" value of its type. It's not proper IMO, but it's also outside the scope of my PR to fix

Reply:
#7189